geospaas/viewer/templates/viewer/site_base.html

```diff
@@ -1,4 +1,4 @@
-{% load staticfiles %}
+{% load static %}
```

Comment:
Does it still work with Django 2.2?

Reply:
I have built it and it is working on my local machine